charts/openbao/.github/pull_request_template.md

```diff
@@ -0,0 +1,43 @@
+<!--
+    Thank you for your contribution. Please use this template to help guide
+    you towards preparing a PR that fullfills our merge requirements.
+-->
+
+# Description
+
+<!-- Describe the changes your PR introduces here. -->
+
+# Issues
+
+<!--
+    Link related issues here, it's ok if this is empty but we do recommend that
+    you create issues before working on PRs, issues on internal trackers are
+    fine and need not be linked here.
+
+    If your PR adds a new chart we expect you to create and link an issue so we
+    can discuss adding your chart before you put the work into creating it.
+-->
+
+# Checklist
+
+<!--
+    Take care of the default items before marking your PR as ready for review,
+    be prepared to add more items.
+-->
+
+- [ ] This PR contains a description of the changes I'm making
+- [ ] I read the `CONTRIBUTING.md` guide
+- [ ] I updated the version in Chart.yaml if feasable.
```

Comment:
We could link to [semver.org](https://semver.org/) as info about the version scheme